.github/workflows/recipe-build.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish Docker Image
+
+on:
+  push:
+    branches: [ main ]
+  release:
+    types:
+      - published
+        
+jobs:
+
+  build:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+        
+      - name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
+        with:
+          images: ghcr.io/Gordiancyber/recipe-stable-diffusion-2-1
```

Comment:
Why is this other repo included in this salad repo script?

Reply:
Its was a reference while writing that. I’ll effect changes and make another PR for it now.

Reply:
Also, this repo for stable diffusion would exist in the recipe repo as a part of it?